==33270== Invalid read of size 4
==33270==    at 0x10A9F2: actualEnIterador(_rep_iterador*) (iterador.cpp:76)
==33270==    by 0x10ABDB: reversoDeIterador(_rep_iterador*) (aplicaciones.cpp:55)
==33270==    by 0x10953D: main (principal.cpp:94)
==33270==  Address 0x0 is not stack'd, malloc'd or (recently) free'd
==33270== 
==33270== 
==33270== Process terminating with default action of signal 11 (SIGSEGV)
==33270==  Access not within mapped region at address 0x0
==33270==    at 0x10A9F2: actualEnIterador(_rep_iterador*) (iterador.cpp:76)
==33270==    by 0x10ABDB: reversoDeIterador(_rep_iterador*) (aplicaciones.cpp:55)
==33270==    by 0x10953D: main (principal.cpp:94)
==33270==  If you believe this happened as a result of a stack
==33270==  overflow in your program's main thread (unlikely but
==33270==  possible), you can try to increase the size of the
==33270==  main thread stack using the --main-stacksize= flag.
==33270==  The main thread stack size used in this run was 8388608.
1>#  implementar reversoDeIterador.
2>timeout: la orden monitorizada ha volcado un 'core'
Segmentation fault
